### DISCUSSION TITLE
Can you group processes by executable?

### DISCUSSION THREAD

**Discussion by Slopz** [winsiderss/systeminformer#1780] [Questions / Answers] [answered]
I'm not sure if this is a feature since I can't seem to find an option for it, but is it possible to group processes by executable like Task Manager can? Thanks.

**Comment by Vasilich** [comment, marked as answer]
you can group processes by parent - this is much better than TaskManager can do. At least here locally TaskManager doesn't group all instances of Firefox, but SI does:
![image](https://github.com/winsiderss/systeminformer/assets/1168434/6d57aecc-a301-4fcb-9370-9cae40ae16b8)

And SI even can show summary CPU, RAM and IO bytes for all these collapsed processes:
![image](https://github.com/winsiderss/systeminformer/assets/1168434/340cb40a-86c9-4cec-8b5b-d0302cda4971)

**Comment by Slopz** [comment]
This is pretty much what I'm looking for...but how do I do it? I can't seem to find the option for it. 

Edit: clicking on the name column seems to do it...but then I can't sort by CPU usage while still maintaining the grouping which is what I'd want...oh well.

**Comment by Vasilich** [comment]
No, SI can't do sorting inside groups.
Just as a possible workaround - you can filter by exe name using the search box, and then sort resulting filtered list to any column:
![image](https://github.com/winsiderss/systeminformer/assets/1168434/0dd952ca-6f76-4bdc-b998-f2eb32bc4eef)
